vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/validation/BaseValidations.java

```diff
@@ -100,6 +100,13 @@ public boolean assertElementNumber(String businessDescription, String systemDesc
                 elementNumber(Matchers.comparesEqualTo(number)));
     }
 
+    @Override
+    public boolean assertElementsNumber(String description, List<WebElement> elements, int number)
```

Comment:
what's the reason for this method?

Reply:
I'll need this in my next PR, additionally it's replacement for a deprecated method